src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesV1Spec.scala

```diff
@@ -23,6 +23,7 @@ import scala.collection.concurrent.TrieMap
 /**
   * Created by dvoet on 6/7/17.
   */
+@deprecated("this allows testing of deprecated functions, remove as part of CA-1031", "")
```

Comment:
Should this be duplicated for `ResourceRoutesSpec`? Or will we leave the unversioned resource routes and just have them start pointing to v2 going forward? I had always thought of them as synonymous with v1, but I'm realizing that's not necessarily true

Reply:
we will remove `ResourceRoutesSpec` but it did not use deprecated functions